content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,133 @@
+{{% /alert %}}
+
+{{% alert note %}}
+*Category 2*
+
+Writing blogs about KubeEdge either in the [website](https://github.com/kubeedge/website) (on PR approved & merged, this gets published in kubeedge.io website blog) or in other popular technical blogging site.
+
+Submit your blog details  [here](https://docs.google.com/forms/d/e/1FAIpQLSd9luGaHWt5jABEQjtr11CgWLVw7Px6tJf0TIjidbl3UL_Izg/viewform).
+
+{{% /alert %}}
+
+{{% alert note %}}
+*Category 3*
+
+Create your own sample applications and demo examples to illustrate possible use case(s) of using KubeEdge in repo [examples](https://github.com/kubeedge/examples).
+{{% /alert %}}
+
+
+## **How the winners are selected**
+
+{{% alert note %}}
+In case of Category 1
+
+KubeEdge maintainers will review the code submitted with PR(s) for feature / test case development or issue fix. 
+For issues raised, KubeEdge maintainers will review severity of issue, quality of steps described towards reproducing, data sufficiency. 
+If a requirement is logged under issues, then maintainers will review against quality of description, uniqueness & how much value it could create in comparision with other edge computing platforms.
+
+One individual from this category will win free pass to free pass to [KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019.](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019/)
+{{% /alert %}}
+
+
+{{% alert note %}}
+In case of Category 2 
+
+KubeEdge maintainers will decide winner based on the content & popularity of that blog. 
+
+One individual from this category will win free pass to free pass to [KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019.](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019/)
+{{% /alert %}}
+
+{{% alert note %}}
+In case of Category 3 
+
+KubeEdge maintainers will review the code, documentation describing the steps & how to use guide submitted with PR(s) submitted in repo examples.
+
+One individual from this category will win free pass to free pass to [KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019.](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019/)
+{{% /alert %}}
+
+## **Who can participate ?**
+
+As majority of KubeEdge contributors are currently from Huawei organization, KubeEdge maintainers want to encourage other individuals, hence any commits done from Github accounts which are based on huawei domain will not qualify in participation. Other than non-huawei domain all other commits are considered. Hence this contest is open for any individuals.
+
+
+## **Timeline**
+From 20th April 2019 to 23rd May 2019
+
+
+## **How the winners are notified ?**
+
+In this blog a “Result” section will be added on 31st May 2019. Winners Github ID will be published in this section. An e-mail will also be sent to winners respective e-mail ID’s associated with Github ID. In case of any clarification required on results one could reach out to us via
```

Comment:
Can't we announce on 23rd ? in Kubecon barcelona itself ?

Reply:
I would like to give people more time to deeply get involved in this activity.

Reply:
As discussed addressed it.